File: CpogsPlugin/src/main/java/org/workcraft/plugins/cpog/ControlPoint.java
package org.workcraft.plugins.cpog;

import java.awt.geom.Point2D;

import org.workcraft.dependencymanager.advanced.user.ModifiableExpression;

public class ControlPoint {

	public ControlPoint(ModifiableExpression<Point2D> position) {
		this.position = position;
	}

	public final ModifiableExpression<Point2D> position;
}

File: CpogsPlugin/src/main/scala/org/workcraft/plugins/cpog/scala/ControlPoint.scala
package org.workcraft.plugins.cpog.scala
import java.awt.geom.Point2D
import nodes._
import org.workcraft.scala.Expressions._
import org.workcraft.dom.visual.BoundedColorisableGraphicalContent
import org.workcraft.dom.visual.Touchable
import org.workcraft.graphics.RichGraphicalContent

class ControlPoint (val position: ModifiableExpression[Point2D.Double], val graphics:Expression[RichGraphicalContent])